taskvine/src/examples/vine_example_peer_transfer.c

```diff
@@ -24,58 +12,51 @@ with all the same tasks on the worker.
 #include <errno.h>
 #include <unistd.h>
 
-const char *query_string = ">P01013 GENE X PROTEIN (OVALBUMIN-RELATED)\n\
-		QIKDLLVSSSTDLDTTLVLVNAIYFKGMWKTAFNAEDTREMPFHVTKQESKPVQMMCMNNSFNVATLPAE\n\
-		KMKILELPFASGDLSMLVLLPDEVSDLERIEKTINFEKLTEWTNPNTMEKRRVKVYLPQMKIEEKYNLTS\n\
-		VLMALGMTDLFIPSANLTGISSAESLKISQAVHGAFMELSEDGIEMAGSTGVIEDIKHSPESEQFRADHP\n\
-		FLFLIKHNPTNTIVYFGRYWSP\n\
-";
-
-#define BLAST_URL "https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-2.13.0+-x64-linux.tar.gz"
+#define BLAST_URL "https://mirrors.slackware.com/slackware/slackware-iso/slackware-12.0-iso/slackware-12.0-install-d1.iso"
 
 #define LANDMARK_URL "https://ftp.ncbi.nlm.nih.gov/blast/db/landmark.tar.gz"
```

Comment:
Maybe remove this one? We can make the example targeted to the peers sharing one big file.